src/reconcile/rec_write.c

```diff
@@ -3286,6 +3285,17 @@ __rec_split_write(WT_SESSION_IMPL *session,
 		}
 	}
 
+	bnd->entries = r->entries;
+	/* Output a verbose message if we create a page without many entries */
+	if (WT_VERBOSE_ISSET(session, WT_VERB_SPLIT) && r->entries < 6)
+		WT_ERR(__wt_verbose(session, WT_VERB_SPLIT,
+		    "Reconciliation creating a page with %" PRIu32
+		    " entries, memory footprint %" PRIu64
+		    ", page count %" PRIu32 ", %s, split state: %d\n",
+		    r->entries, r->page->memory_footprint, r->bnd_next,
+		    F_ISSET(r, WT_EVICTING) ? "evict" : "checkpoint",
+		    r->bnd_state));
+
```

Comment:
I'm not sure about this verbose message: if it's been useful to you, I'm good with it, but if it's not been sufficiently helpful, I'd remove it (or ifdef it out). The reason is if you have a small page size (like a lot of format runs do), most created pages will have less than 6 entries, which means all we'll see is these messages.

Alternatively, maybe we could do something like:
```
(r->split_size / r->entries > 512)
```
but I don't know if that's sufficient for what you were trying to debug.

I'm also curious about the memory footprint and eviction/checkpoint distinction. When you were debugging this one, how did those end up being useful? (I have no objection to them being there, just curious why they were useful to you.)